inductiva/simulators/simulator.py

```diff
@@ -6,12 +6,30 @@
 from inductiva.utils import files
 
 
+def mpi_enabled(cls):
+    """Class decorator that adds MPICluster to the supported resources.
+    """
+    supports = getattr(cls, "_supported_resources", None)
+    cls._supported_resources = supports | {resources.MPICluster}  # pylint: disable=protected-access
```

Comment:
The default in getattr shouldn't be None, because None isn't a supported resource.
At most, it could be `type(None)`, but `NoneType` isn't a supported resource.

```suggestion
    supports = getattr(cls, "_supported_resources", set())
    cls._supported_resources = supports | {resources.MPICluster}  # pylint: disable=protected-access
```